docs/auth/social-auth.md

```diff
@@ -77,6 +77,38 @@ export const googleLogin = async () => {
 ```
 
 ## Twitter
+You can use react-native-twitter-signin for Twitter authentication. This library handles the flow of logging in a user and accesssing their `authToken` and `authTokenSecrent` by wrapping around the official Twitter library.
+
+The`react-native-twitter-signin` library allows us to login (using `RNTwitterSignIn`) which returns `authToken` and `authTokenSecret`
+
+Make sure you thorougly follow the instructions on setting up the necessary requirements to get react-native-twitter-signin working.
+
+
+```js
+import {NativeModules} from 'react-native}
```

Comment:
```suggestion
import {NativeModules} from 'react-native}';
```

Reply:
There's a random bracket at the end, it should be

import { NativeModules } from 'react-native';

Reply:
Updated